Sources/Networking/Operations/GetOfferingsOperation.swift

```diff
@@ -59,8 +59,9 @@ private extension GetOfferingsOperation {
                     return statusCode.isSuccessfulResponse
                     ? .success(response)
                     : .failure(
-                        ErrorUtils.backendError(withBackendCode: BackendErrorCode(code: response["code"]),
-                                                backendMessage: response["message"] as? String)
+                        ErrorResponse
```

Comment:
This is SO much better. I hacked on the errors a bunch just to get it working and surfacing what we wanted to surface, but this is a nice API 🧡